lib/customer.rb

```diff
@@ -34,4 +32,16 @@ def update_last_name(name)
   def update_time
     @updated_at = Time.now
   end
+
+  def created_at
+    return @created_at if @created_at.instance_of?(Time)
+
```

Comment:
Looks great!! 😊  (Also reminder for later: possibly will remove extra line here)

Reply:
Good reminder! Making that update